framework/src/modules/chain/submodules/blocks/process.js

```diff
@@ -19,6 +19,7 @@ const async = require('async');
 const { Status: TransactionStatus } = require('@liskhq/lisk-transactions');
 const slots = require('../../helpers/slots');
 const definitions = require('../../schema/definitions');
+const InitTrs = require('../../logic/init_transaction');
```

Comment:
InitTransaction is passed through via scope to the constructor function in the rest of the application. We should follow the same practice.